tests/test_unit/test_docs/test_build.py

```diff
@@ -0,0 +1,188 @@
+import json
+import logging
+from pathlib import Path
+from textwrap import dedent
+
+import pydantic
+import pytest
+
+from docs.build import build_schemas_from_source, dump_json_schema
+
+
+def test_dump_json_schema(tmp_path):
```

Comment:
Don't we have exactly this same test under `tests/test_unit/test_docs/test_build.py`?

Reply:
I believe the same applies to a couple of tests

Reply:
uh, must have moved that file and gotten mixed up.
Good catch! Thanks